compose/local/django/Dockerfile

```diff
@@ -1,55 +1,30 @@
-# define an alias for the specfic python version used in this file.
-FROM python:3.10.10-slim-bullseye as python
+FROM python:3.10.10-slim-bullseye
 
-# Python build stage
-FROM python as python-build-stage
-
-ARG BUILD_ENVIRONMENT=local
-
-# Install apt packages
-RUN apt-get update && apt-get install --no-install-recommends -y \
-  # dependencies for building Python packages
-  build-essential \
-  # psycopg2 dependencies
-  libpq-dev
-
-# Requirements are installed here to ensure they will be cached.
-COPY ./requirements .
-
-# Create Python Dependency and Sub-Dependency Wheels.
-RUN pip wheel --wheel-dir /usr/src/app/wheels  \
-  -r ${BUILD_ENVIRONMENT}.txt
-
-
-# Python 'run' stage
-FROM python as python-run-stage
-
-ARG BUILD_ENVIRONMENT=local
 ARG APP_HOME=/app
+WORKDIR ${APP_HOME}
 
-ENV PYTHONUNBUFFERED 1
-ENV PYTHONDONTWRITEBYTECODE 1
-ENV BUILD_ENV ${BUILD_ENVIRONMENT}
 
-WORKDIR ${APP_HOME}
+ARG BUILD_ENVIRONMENT=local
+ENV PATH="/root/.local/bin:${PATH}" \
+    POETRY_VIRTUALENVS_CREATE=false \
+    PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    BUILD_ENV=${BUILD_ENVIRONMENT}
 
-# Install required system dependencies
 RUN apt-get update && apt-get install --no-install-recommends -y \
-  # psycopg2 dependencies
-  libpq-dev \
-  # Translations dependencies
-  gettext \
```

Comment:
are you sure w don't need those?

Reply:
we need those, they are within this line https://github.com/SCOAP3/scoap3/blob/84341ce9912b22012e59c754da2f4fe3fc11e118/compose/local/django/Dockerfile#L15